### PR TITLE
Fix categories controllers

### PR DESCRIPTION
**Pull request by LesiaM-T**
Fixed CRUD controllers for categories

### CODE REVIEW

File: src/controllers/categories.js [hunk 1]
```diff
@@ -22,17 +22,15 @@ const categoriesByGroup = async (req, res) => {
   res.status(201).json(result);
 };
 
-
-// TODO feat controllers
 const createCategory = async (req, res) => {
     
-  const result = await Category.create(req.body);
+  const result = await Category.create(req.query);
```
Comment (VadymKharyton) [inline src/controllers/categories.js:27]: The previous solution with req.body was better, because in POST methods we should pass data in body

Comment (LesiaM-T) [reply src/controllers/categories.js:27]: But it was not working with "body" and with "query" works. At list in Postman I was testing

Comment (VadymKharyton) [reply src/controllers/categories.js:27]: It probably didn't work, because you tried to pass data in `Params` tab instead of passing JSON in `Body` tab

Comment (LesiaM-T) [reply src/controllers/categories.js]: Yes, thank you. The mistake was in using wrong tabs in Postman.

**Review by VadymKharyton — APPROVED** [review]

**Pull request merged**